<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2">
     <widget class="QPushButton" name="pB_Get">
      <property name="text">
       <string>Получить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="cB_Request">
      <item>
       <property name="text">
        <string>Все</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Комедии</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ужасы</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" rowspan="4">
     <widget class="QTableView" name="tbV_tableResult"/>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="lb_Status">
      <property name="text">
       <string>Неподключено</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
